jsons/1.12.1-rel.json

```diff
@@ -38,9 +38,28 @@
       "clientreq":true
     },
     {
-      "name": "jline:jline:2.13",
+      "name": "net.minecraftforge:log4j-server-fix:2.8.1",
```

Comment:
Needs to be built using https://gist.github.com/Minecrell/0d4ccfab75eee044a654b196e61c93c8 (put it in a directory called `log4j-server-fix`, then run `gradle build`) and uploaded to the Forge maven to make this work.